libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/availableValueFilter.spec.ts

```diff
@@ -60,4 +68,61 @@ describe("Available value filter", () => {
             .elementsAreLoaded()
             .hasFilterListSize(1);
     });
+
+    it("should extend attribute filter by date filter", { tags: "checklist_integrated_tiger" }, () => {
+        Navigation.visit("dashboard/multiple-date-filters", featureFlags);
+        cy.intercept("GET", "**/attributes**").as("attributes");
+
+        cy.wait("@attributes").then(() => {
+            const dateFilter = new DateFilter();
+            const dateFilterActivity = new DateFilter("Activity");
+            const dateFilterAbsoluteForm = new DateFilterAbsoluteForm();
+            topBar.enterEditMode().editButtonIsVisible(false);
+
+            dateFilter.openAndSelectOption(".s-absolute-form");
+            dateFilterAbsoluteForm
+                .typeIntoFromRangePickerInput("4/3/2010")
+                .typeIntoToRangePickerInput("4/3/2018");
+            dateFilter.pressButton("apply");
+
+            dateFilterActivity.openAndSelectOption(".s-absolute-form");
+            dateFilterAbsoluteForm
+                .typeIntoFromRangePickerInput("4/3/2010")
+                .typeIntoToRangePickerInput("4/3/2018");
+            dateFilterActivity.pressButton("apply");
+        });
+
+        widget
+            .waitChartLoaded()
+            .getChart()
+            .getDataLabelValues()
+            .should("deep.equal", ["$4,108,360.80", "$2,267,528.48", "$3,461,373.87"]);
+
+        salesRepFilter
+            .open()
+            .elementsAreLoaded()
+            .hasFilterListSize(22)
+            .selectConfiguration()
+            .configureLimitingDateFilterDependency("activity", "Date range")
+            .hasFilterListSize(20);
+
+        salesRepFilter
+            .elementsAreLoaded()
+            .selectConfiguration()
+            .isSpecificDateFilterVisible("activity", false)
```

Comment:
After saleReps filter is selectd at 106 line, then isSpecificDateFilterVisible = false, it seems inconsistent?

Reply:
It's is correct, because when choose Date range (common date) , Activity Filter (specific date) will be disabled